hall_converter:
  ros__parameters:
    l_gradient: 0.0
    l_offset: 0.0
    left_topic: /hall/left
    r_gradient: 0.0
    r_offset: 0.0
    right_topic: /hall/right
